cmd/kots/cli/install.go

```diff
@@ -745,3 +777,173 @@ func CheckRBAC() error {
 	}
 	return nil
 }
+
+func ValidateAutomatedInstall(deployOptions kotsadmtypes.DeployOptions, authSlug string, apiEndpoint string) error {
+	url := fmt.Sprintf("%s/app/%s/automated/status", apiEndpoint, deployOptions.License.Spec.AppSlug)
+
+	startTime := time.Now()
+
+	for time.Since(startTime) < deployOptions.Timeout {
+		taskStatus, err := getAutomatedInstallStatus(url, authSlug)
+		if err != nil {
+			return errors.Wrap(err, "failed to get automated install status")
+		}
+		switch taskStatus.Status {
+		case automation.AutomatedInstallFailed:
+			return errors.New(taskStatus.Message)
+		case automation.AutomatedInstallSuccess:
+			return nil
+		}
+		time.Sleep(time.Second)
+	}
+
+	return errors.New("timeout waiting for automated install. Use the --wait-duration flag to increase timeout.")
+}
+
+func getAutomatedInstallStatus(url string, authSlug string) (*kotsstore.TaskStatus, error) {
+	newReq, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create request")
+	}
+	newReq.Header.Add("Content-Type", "application/json")
+	newReq.Header.Add("Authorization", authSlug)
+
+	resp, err := http.DefaultClient.Do(newReq)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to execute request")
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 200 {
```

Comment:
```suggestion
	if resp.StatusCode != http.StatusOK {
```

Reply:
updated. spaced using the "commit suggestion" button